FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m

```diff
@@ -30,8 +29,8 @@ - (void)runInitialization {
         && self.partners.count > 0
         && [self.partners.firstObject app_id]) {
 
-        OguryConfigurationBuilder *configurationBuilder = [[OguryConfigurationBuilder alloc] initWithAssetKey:[self.partners.firstObject app_id]];
-        [Ogury startWithConfiguration:[configurationBuilder build]];
+        [OguryChoiceManager.sharedManager setupWithAssetKey:[self.partners.firstObject app_id]];
+        [OguryAds.shared setupWithAssetKey:[self.partners.firstObject app_id]];
```

Comment:
@carlosalcala-freestar  do u get deprecating warnings?  if so, please use the latest ogury APIs... thx.

Reply:
Fixing those @maddapper